docker-compose.yml

```diff
@@ -33,27 +35,40 @@ services:
     environment:
       DATALAND_SKYMINDERCLIENT_TOKEN: ${DATALAND_SKYMINDERCLIENT_TOKEN}
       DATALAND_SKYMINDERCLIENT_USER: ${DATALAND_SKYMINDERCLIENT_USER}
+      DATALAND_EDC_TOKEN: ${DATALAND_EDC_TOKEN}
+      DATALAND_EDC_USER: ${DATALAND_EDC_USER}
   frontend:
     build:
       context: ./
       dockerfile: ./dataland-frontend/Dockerfile
+      args:
+        DATALAND_EDC_TOKEN: ${DATALAND_EDC_TOKEN}
+        DATALAND_EDC_USER: ${DATALAND_EDC_USER}
     volumes:
       - ./dataland-frontend/default.conf:/etc/nginx/conf.d/default.conf
     expose:
       - "80"
     networks:
       - internal
-  skyminder-server:
+  skyminder-dummyserver:
     build:
       context: ./
-      dockerfile: ./dataland-skyminder-server/Dockerfile
+      dockerfile: ./dataland-skyminder-dummyserver/Dockerfile
     networks:
       - internal
+      - external
     expose:
       - "8080"
     environment:
       DATALAND_SKYMINDERCLIENT_TOKEN: ${DATALAND_SKYMINDERCLIENT_TOKEN}
       DATALAND_SKYMINDERCLIENT_USER: ${DATALAND_SKYMINDERCLIENT_USER}
+  dataland-edc:
+    image: ghcr.io/d-fine/datalandedc/edcdummyserver:latest
```

Comment:
Eine Möglichkeit ist auch, einen Alias zu benutzen. Bin mir nicht sicher ob ich das besser finden:
statt:

```
  dataland-edc:
    image: ghcr.io/d-fine/datalandedc/edcdummyserver:latest
```
dann:
```
  dataland-edc-dummyserver:
    alias: dataland-edc
    image: ghcr.io/d-fine/datalandedc/edcdummyserver:latest
```

Was meint ihr?

Reply:
FInde ich gut.

Reply:
Umgesetzt, allerdings musst man wie folgt die Aliases pro Network definieren:

![image](https://user-images.githubusercontent.com/99255296/161585873-433520c7-721a-4145-bd8b-16e7d5e87d11.png)

Reply:
Ich habe den service mal in dataland-edc-dummyservice umbenannt ansonsten ergibt das mit dem alias ja wenig Sinn wenn die alle gleich heißen

Reply:
ok